cmd/gossamer/main.go

```diff
@@ -174,15 +174,23 @@ func nodeInitialized(ctx *cli.Context) bool {
 	registry := path.Join(cfg.Global.DataDir, "KEYREGISTRY")
 	_, err = os.Stat(registry)
 	if os.IsNotExist(err) {
-		log.Error("[gossamer] Failed to locate KEYREGISTRY", "error", err)
+		log.Warn(
+			"[gossamer] Node has not been initialized",
+			"datadir", cfg.Global.DataDir,
+			"error", "failed to locate KEYREGISTRY file in data directory",
+		)
 		return false
 	}
 
 	// check if manifest exists
 	manifest := path.Join(cfg.Global.DataDir, "MANIFEST")
 	_, err = os.Stat(manifest)
 	if os.IsNotExist(err) {
-		log.Error("[gossamer] Failed to locate MANIFEST", "error", err)
+		log.Warn(
```

Comment:
should we be warning if the node isn't initialized? or rather, if the node isn't initialized and we warn, we should also log that we initialized it for them

Reply:
the warning is warning them that we are initializing a node for them, I can clarify in warning